src/command_manager.cc

```diff
@@ -36,7 +36,9 @@ using namespace com::centreon::engine::downtimes;
 /**
  *  The default constructor
  */
-command_manager::command_manager() {}
+command_manager::command_manager() {
+  _has_data = false;
+}
```

Comment:
Bad class initialization...

command_manager::command_manager() : _has_data{false} {}

is better.